app/models/concerns/umrdr/generic_work_metadata.rb

```diff
@@ -4,6 +4,9 @@ module GenericWorkMetadata
     extend ActiveSupport::Concern
     included do
       property :subject, predicate: ::RDF::Vocab::MODS.subject
+      property :date_coverage, predicate: ::RDF::Vocab::DC.coverage do |index|
+        index.as :stored_searchable, :facetable
```

Comment:
We actually shouldn't index this as a string.  I don't think the serialization isn't going to produce meaningful search results for anyone without verbatim matches.  I was looking at: https://cwiki.apache.org/confluence/display/solr/Working+with+Dates, but haven't gotten as far as figuring out how to do that in ActiveFedora.